html_css/css_flexbox.md

```diff
@@ -0,0 +1,27 @@
+### Introduction
+
+The Flexible Box Module, usually referred to as flexbox, was designed as a one-dimensional layout model, and as a method that 
+could offer space distribution between items in an interface and powerful alignment capabilities. This article gives an outline
+of the main features of flexbox, which we will be exploring in more detail in the rest of these guides.
+
+In this section, you'll get a look at how to use CSS Flex Box to make your projects more awesome!.
+
+### Learning Outcomes
+* What is CSS flex?
+* Why are flex effective for helping site layout?
+* What are the major different types of flex in use today?
```

Comment:
Are there different versions? I wasn't aware of this.

Reply:
No it was just a typo error sorry there is no other kind of flexbox. @KevinMulhern